### PR TITLE
Added 101-servicebus-topic-subscription Template

### PR DESCRIPTION
**Pull request by crpietschmann**
### Changelog
* Added "101-servicebus-topic-subscription" Template

### Description of the change
This template creates a Service Bus Namespace and Topic with a Subscription to catch all messages.

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@crpietschmann__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>You've already signed the contribution license agreement. Thanks!</span>
        <span>We will now validate the agreement and then real humans will evaluate your PR.</span>

TTYL, AZPRBOT;

**Comment by crpietschmann** [comment]
I don't know what the "cannot reference itself" error is about. This ARM Template runs just fine as is on Azure. I have tested it's deployment myself. I assume it's just an issue with the Parameters all having their values set to "GEN-UNIQUE" so they work with the Template infrastructure. These aren't the values that would be deployed on Azure, so I'm not sure if there is anything to fix.

Please let me know if I do need to fix something. Thanks!

File: 101-servicebus-topic-subscription/README.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# Create a Service Bus Namespace
+
+<a href="https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-quickstart-templates%2Fmaster%2F101-servicebus-topic-subscription%2Fazuredeploy.json" target="_blank">
+    <img src="http://azuredeploy.net/deploybutton.png"/>
+</a>
+
+This template creates a Service Bus Namespace and Topic with a Subscription to catch all messages.
```
Comment (singhkay) [inline 101-servicebus-topic-subscription/README.md]: More documentation would be good

**Comment by singhkay** [comment]
@crpietschmann Don't worry about the acomghbot errors for now. There is a fix needed. Thank you for your contribution! lgtm, merged

**Comment by crpietschmann** [comment]
Awesome! The PR merge process for this project is so smooth! Great Job!

**Comment by singhkay** [comment]
@crpietschmann Appreciate the kind words :)

**Pull request merged**